.github/workflows/release-pr.yaml

```diff
@@ -2,11 +2,11 @@ name: create_release_pull_request
 on:
   push:
     tags:
-      - 'v[0-9]+.[0-9]+.0' # run this workflow when a new minor version is published
+      - "v[0-9]+.[0-9]+.0" # run this workflow when a new minor version is published
```

Comment:
Why is this being modified? just curious.

Reply:
probably my linter did it :)